django/db/models/indexes.py

```diff
@@ -73,7 +74,7 @@ def remove_sql(self, model, schema_editor):
     def deconstruct(self):
         path = '%s.%s' % (self.__class__.__module__, self.__class__.__name__)
         path = path.replace('django.db.models.indexes', 'django.db.models')
-        return (path, (), {'fields': self.fields, 'name': self.name})
+        return (path, (), {'fields': self.fields, 'name': self.name, 'db_tablespace': self.db_tablespace})
```

Comment:
Looking at the additional changes that this brought in, thoughts on only adding `db_tablespace` when it's set?